partitura/io/importmidi.py

```diff
@@ -287,18 +284,42 @@ def load_performance_midi(
                 time_signatures=time_signatures,
                 meta_other=meta_other,
                 ppq=ppq,
-                mpq=mpq,
+                mpq=default_mpq,
                 track=i,
             )
 
             pps.append(pp)
+    
+    # adjust timing of events based on tempo changes
+    for pp in pps:
+        for note in pp.notes:
+            note["note_on"] = adjust_time(note["note_on_tick"], tempo_changes, ppq)
+            note["note_off"] = adjust_time(note["note_off_tick"], tempo_changes, ppq)
+        for control in pp.controls:
+            control["time"] = adjust_time(control["time_tick"], tempo_changes, ppq)
+        for program in pp.programs:
+            program["time"] = adjust_time(program["time_tick"], tempo_changes, ppq)
```

Comment:
Should we also adjust the time of `time_signatures`, `key_signatures` and `meta_other`?